src/DashboardHub/Bundle/AppBundle/Controller/DashboardController.php

```diff
@@ -10,8 +14,82 @@
  */
 class DashboardController extends Controller
 {
+
+    /**
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
     public function indexAction()
     {
-        return $this->render('DashboardHubAppBundle:Dashboard:index.html.twig');
+        return $this->render(
+                'DashboardHubAppBundle:Dashboard:index.html.twig',
+                array(
+                    'dashboards' => $this->get('dashboardhub_app_main.service.dashboard')
+                        ->findAllByAuthenticatedUserAndDefaults()
+                )
+            );
+    }
+
+    public function addAction(Request $request)
```

Comment:
Fixed in next PR https://github.com/DashboardHub/PipelineDashboard/pull/64/files#diff-cbf79bbb80822cc4f7d69acaff41f11aR37